### PR TITLE
Migration of logoScreenSaver and playAccessValidation to ES6 module

### PR DESCRIPTION
**Pull request by Camc314**
**Issues**
part of #868

### CODE REVIEW

**Comment by dkanada** [comment]
Could you fix the linting errors on master in this pull request?

**Comment by Camc314** [comment]
Yup will do.

**Comment by MrTimscampi** [comment]
@dkanada They're fixed as part of #1675, afaik

File: src/plugins/logoScreensaver/plugin.js [hunk 1]
```diff
@@ -1,165 +1,165 @@
...
-    return function () {
-        var self = this;
+import pluginManager from 'pluginManager';
 
-        self.name = 'Logo ScreenSaver';
-        self.type = 'screensaver';
-        self.id = 'logoscreensaver';
-        self.supportsAnonymous = true;
+export default function () {
+    const self = this;
```
Comment (grafixeyehero) [inline src/plugins/logoScreensaver/plugin.js:4]: You can remove this alias.

**Pull request merged**